src/js/communities/community-data.js

```diff
@@ -0,0 +1,95 @@
+export const CommunityData = [{
+    name: 'Newcastle Coders Group',
+    location: "ICT Building, University of Newcastle",
+    period: {
+        interval: "Monthly",
+        //maybe use a single digit here and convert it later (i.e. "2" becomes "2nd")
```

Comment:
We can probably remove this as to not cause confusion, I am guessing this is just train of thought.

Reply:
Oh sorry, I thought I removed that comment.